src/Test/test_infeasibility_certificates.jl

```diff
@@ -0,0 +1,53 @@
+for sense in (MOI.MIN_SENSE, MOI.MAX_SENSE), offset in (0.0, 1.2)
+    o = iszero(offset) ? "" : "_offset"
+    f_name = Symbol("test_unbounded_$(Symbol(sense))$(o)")
+    @eval begin
+        function $(f_name)(model::MOI.ModelLike, config::Config)
+            @requires _supports(config, MOI.optimize!)
+            x = MOI.add_variable(model)
+            MOI.set(model, MOI.ObjectiveSense(), $sense)
+            f = 2.2 * x + $offset
+            MOI.set(model, MOI.ObjectiveFunction{typeof(f)}(), f)
+            MOI.optimize!(model)
+            @requires(
+                MOI.get(model, MOI.TerminationStatus()) == MOI.DUAL_INFEASIBLE,
+            )
+            @requires(
+                MOI.get(model, MOI.PrimalStatus()) ==
+                MOI.INFEASIBILITY_CERTIFICATE,
+            )
+            d = MOI.get(model, MOI.VariablePrimal(), x)
+            if $sense == MOI.MIN_SENSE
+                @test d < -config.atol
+            else
+                @test d > config.atol
+            end
+            @test(
+                isapprox(2.2 * d, MOI.get(model, MOI.ObjectiveValue()), config),
```

Comment:
@blegat, we've obviously never tested or tried this. GLPK and Gurobi both fail the ObjectiveValue test.